weasyl/controllers/moderation.py

```diff
@@ -111,91 +136,83 @@ def modcontrol_massaction_(request):
         content_type='text/plain',
         body=moderation.bulk_edit(
             request.userid,
-            form.action,
-            map(int, form.submissions),
-            map(int, form.characters),
-            map(int, form.journals),
+            action,
+            map(int, submissions),
+            map(int, characters),
+            map(int, journals),
         ),
     )
 
 
 @moderator_only
 @token_checked
 def modcontrol_hide_(request):
-    form = request.web_input(name="", submission="", character="")
-
-    if form.submission:
-        moderation.hidesubmission(int(form.submission))
-    elif form.character:
-        moderation.hidecharacter(int(form.character))
+    if request.params.get('submission'):
+        moderation.hidesubmission(int(request.params.get('submission')))
```

Comment:
When it’s required (e.g. here, `int(None)` is an error), you can use bracket notation:

```suggestion
        moderation.hidesubmission(int(request.params['submission']))
```

(applies to other instances too)